viral/fix_tiff.py

```
@@ -0,0 +1,130 @@
+    tiff_read = tifffile.imread(tiff_path, key=range(0, n_frames - 1, 1))
+    output_path = tiff_path.parent / (tiff_path.stem + "_until_broken.tiff")
+    tifffile.imwrite(output_path, tiff_read, bigtiff=True)
+    print("Done")
+
+
+def check_if_tiff_is_broken(tiff_path: Path) -> bool:
+    """Checks whether a tiff file is broken."""
+    # Doing this with ScanImageTiffReader first as it is faster than tifffile
+    print(f"Checking {tiff_path}")
+    try:
+        tiff_file = ScanImageTiffReader(str(tiff_path))
```

Ruff (0.13.1)

Local variable `tiff_file` is assigned to but never used

Remove assignment to unused variable `tiff_file`

(F841)